app/controllers/old_nodes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class OldNodesController < ApplicationController
+  layout :map_layout
+
+  before_action :authorize_web
+  before_action :set_locale
+  before_action -> { check_database_readable(:need_api => true) }
+  before_action :require_oauth
+
+  authorize_resource
+
+  around_action :web_timeout
+
+  def show
+    @type = "node"
+    @feature = OldNode.preload(:old_tags, :changeset => [:changeset_tags, :user]).find([params[:id], params[:version]])
```

Comment:
In the `browse#node` action for the current view we preload the containing relations here - is that not possible for the historical objects? We still seem to be rendering them?

Reply:
No, we are not rendering them for historic elements.

https://github.com/openstreetmap/openstreetmap-website/blob/6163b01f146d97201e0c2d9efff83dd5dbadebf7/app/models/old_node.rb#L98-L101

Reply:
Ah I had missing that the method didn't actually return anything!